utils/deploy.sh

```
@@ -217,15 +199,13 @@ create_sync_target ${GLBC_SYNC_TARGET_NAME}
 ############################################################
 
 ## Bind to compute APIs
-create_api_binding "kubernetes" "kubernetes" "${GLBC_WORKSPACE}"
+${KUSTOMIZE_BIN} build ${KCP_GLBC_KUSTOMIZATION_DIR}/apiexports/kubernetes | kubectl apply -f -
 
 ## Register GLBC APIs with KCP
 ${KUSTOMIZE_BIN} build ${KCP_GLBC_KUSTOMIZATION_DIR}/kcp-contrib | kubectl apply -f -
 
 ## Register CertManager APIs with KCP
-${KUSTOMIZE_BIN} build ${CERT_MANAGER_KUSTOMIZATION_DIR}/kcp-contrib | kubectl apply -f -
-
-create_api_binding "cert-manager" "cert-manager-stable" "${GLBC_WORKSPACE}"
+${KUSTOMIZE_BIN} build --reorder none ${CERT_MANAGER_KUSTOMIZATION_DIR}/kcp-contrib | kubectl apply -f -
 
 ############################################################
 # Setup GLBC APIExport                                     #
```

@mikenairn Would it make sense to only run the `create_glbc_api_export` script when the `DEPLOY_COMPONENTS` is not `none`?
We could remove the `exit 0` and let it fail gracefully if the APIBinding and APIExport can't be created after the sync targets are set up.

---

I suppose it might be better for it to actually fail if you are expecting it to install something and it can't. Having it exit with a non zero exit code if `DEPLOY_COMPONENTS` isn't `none` probably makes sense, can't think of any reason not to anyway.